Source/bmalloc/libpas/src/libpas/pas_darwin_spi.h

```diff
@@ -31,9 +31,16 @@
 
 #if PAS_OS(DARWIN)
 #if defined(__has_include) && __has_include(<pthread/private.h>)
+
+// FIXME: radar://132606697
+#if !(defined(__has_feature) && __has_feature(modules))
 PAS_BEGIN_EXTERN_C;
+#endif
 #include <pthread/private.h>
+#if !(defined(__has_feature) && __has_feature(modules))
 PAS_END_EXTERN_C;
+#endif
+
```

Comment:
This still feels really weird to me. I think we should conditionally disable the include entirely, rather than import it as extern-C in some places and as C++ in another.

Also, rdar://140431798 is a better radar to associate this with.

Reply:
Yeah, I tend to agree that we should just not #include at all if we can't #include correctly.